pkg/controller/nodes/task/taskexec_context.go

```diff
@@ -140,13 +145,15 @@ func (t *Handler) newTaskExecutionContext(ctx context.Context, nCtx handler.Node
 	}
 
 	resourceNamespacePrefix := pluginCore.ResourceNamespace(t.resourceManager.GetID()).CreateSubNamespace(pluginCore.ResourceNamespace(pluginID))
+	maxAttempts := nCtx.Node().GetRetryStrategy().MinAttempts
```

Comment:
@catalinii This field a bit tricky. Just see how it is used. The tricky part comes in interpretation of this field when you want to attempt 'x' times more or total 'x'

Reply:
It is total maxAttempts according to the code.